.travis.yml

```diff
@@ -7,9 +12,13 @@ cache:
 install:
   - pip install pipenv==8.3.2
   - pipenv install --dev --deploy
+  - npm install -g phantomjs-prebuilt
+  - docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
+  - docker network create rasrmdockerdev_default
```

Comment:
Will this cause the install step to fail if the network already exists (exit code 1)?

Reply:
I don't think it should ever exist in travis because it should spin us up a new instance every time. Part of the reason for putting it in .travis.yml instead of the make file